.github/pull_request_template.md

```diff
@@ -3,56 +3,48 @@
 If this is your first contribution, please review and sign our contributor agreement -
 https://www.elastic.co/contributor-agreement.
 
-Before creating the PR, ensure that:
-
-1. Your branch is rebased on top of the latest master.
```

Comment:
I think we want to retain this guidance

Reply:
I'm going to leave it out to keep the template lean.

It's on everyone to make sure their PR passes tests, will merge cleanly, and that they don't rebase _after_ marking it ready for review; so I think it's implied. We can add it back later if needed.